.circleci/workflows.yml

```
@@ -165,46 +171,16 @@ jobs:
               name: install packages, lint, and create tf template
           - run:
               command: |
-                cd .aws/cdktf.out/stacks/prefect-v2
+                cd .aws/cdktf.out/stacks/<< parameters.stack_name >>
                 terraform init
-                terraform plan
-              name: terraform plan
+                if [[ ${APPLY_STACK} == "false" ]]
+                then
+                  terraform plan
+                else
+                 terraform apply -auto-approve
+                fi
+              name: terraform plan/apply
```

Very minor suggestion: check for 'true' instead of against 'false', to prevent 'False' from being interpreted as 'true'.
```suggestion
                if [[ ${APPLY_STACK} == "true" ]]
                then
                  terraform apply -auto-approve
                else
                  terraform plan
                fi
              name: terraform plan/apply
```